pipeline/schemas/run.py

```diff
@@ -48,6 +43,7 @@ class RunCreate(BaseModel):
     blocking: Optional[bool] = False
     # By default a Run will require GPU resources
     compute_type: ComputeType = ComputeType.gpu
```

Comment:
I think we're planning over having a Run's requirements 'override' those given in the runnable. The `compute_type` field should then also be optional, a missing value meaning 'using the requirements from the runnable'.

Reply:
Good point, have fixed that 👍